include/drjit-core/jit.h

```diff
@@ -1176,6 +1176,10 @@ extern JIT_EXPORT uint32_t jit_var_data(uint32_t index, void **ptr_out);
 /// Query the size of a given variable
 extern JIT_EXPORT size_t jit_var_size(uint32_t index);
 
+/// Query the size of a given variable, as an opaque variable. This also
```

Comment:
I think you can leave the ThreadState notification comment out in `jit.h`, that is true for many other functions as well and we don't list it exhaustively.